var/spack/repos/builtin/packages/neovim/package.py

```diff
@@ -4,37 +4,62 @@
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
 
 
+import os
 from spack import *
 
 
 class Neovim(CMakePackage):
-    """NeoVim: the future of vim"""
+    """Neovim: Vim-fork focused on extensibility and usability"""
 
-    homepage = "http://neovim.io"
-    url      = "https://github.com/neovim/neovim/archive/v0.3.4.tar.gz"
+    homepage = "https://neovim.io"
+    git = "https://github.com/neovim/neovim.git"
+    url = "https://github.com/neovim/neovim/archive/v0.4.3.tar.gz"
+    list_url = "https://github.com/neovim/neovim/releases"
```

Comment:
This shouldn't be needed, `spack versions` and `spack checksum` should work without it. (We actually add this internally for all GitHub packages)